marquez/constants.py

```diff
@@ -0,0 +1,23 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from enum import Enum
+
+
+class JobRunState(Enum):
```

Comment:
Looking over the usage of `JobRunState`, the enum is used to build a path, rather than represent the state returned by an API call on a job object. That is, the states returned by the Marquez API will be `RUNNING`, `COMPLETED`, etc.


I would move the enum to `client.py` or define them inline instead as they represent actions more than states:
```python
    def mark_job_run_running(self, job_run_id, **kwargs):
        return self.mark_job_run_as(
            job_run_id, "run", **kwargs)
```

Reply:
I added another enum along with a mapping. Hopefully that adds a little more formality to the client.

Reply:
Removed the enum.